trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -58,11 +58,12 @@ const getCaseDeal = async (req, res) => {
       deal.tfm.stage = DEAL.DEAL_STAGE.AMENDMENT_IN_PROGRESS;
     }
 
-    const { inProgressPortalAmendments, hasInProgressPortalAmendments } = getAmendmentsInProgress({
-      amendments,
-      deal,
-      teams: req.session.user.teams,
-    });
+    const { inProgressPortalAmendments, hasInProgressPortalAmendments, hasFutureEffectiveDatePortalAmendments, formattedFutureEffectiveDatePortalAmendments } =
+      getAmendmentsInProgress({
+        amendments,
+        deal,
+        teams: req.session.user.teams,
```

Comment:
const { teams } = req.session.user;

```suggestion
        teams,
```